event-grid/create-custom-topic/create-custom-topic.ps1

```diff
@@ -1,15 +1,18 @@
 # Give your custom topic a unique name
-$myTopic = "demoContosoTopic"
+$myTopic = "<your-custom-topic-name>"
+
+# Provide a name for resource group to create. It will contain the custom event.
+$myResourceGroup = "<resource-group-name>"
 
 # Create resource group
-New-AzureRmResourceGroup -Name myResourceGroup -Location westus2
+New-AzureRmResourceGroup -Name $myResourceGroup -Location westus2
 
 # Create custom topic
-New-AzureRmEventGridTopic -ResourceGroupName myResourceGroup -Location westus2 -Name $myTopic
+New-AzureRmEventGridTopic -ResourceGroupName $myResourceGroup -Name $myTopic -Location westus2 
 
 # Retrieve endpoint and key to use when publishing to the topic
-$endpoint = (Get-AzureRmEventGridTopic -ResourceGroupName myResourceGroup -Name $myTopic).Endpoint
-$key = (Get-AzureRmEventGridTopicKey -ResourceGroupName myResourceGroup3 -Name $myTopic).Key1
+$endpoint = (Get-AzureRmEventGridTopic -ResourceGroupName $myResourceGroup -Name $myTopic).Endpoint
+$key = (Get-AzureRmEventGridTopicKey -ResourceGroupName $myResourceGroup3 -Name $myTopic).Key1
```

Comment:
Should be `$myResourceGroup`, not `$myResourceGroup3`.

Reply:
fixed